examples/quick-start-net-core/src/Program.cs

```diff
@@ -0,0 +1,38 @@
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+var serviceName = "tracetest-example";
+var serviceVersion = "1.0.0";
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+
+var appResourceBuilder = ResourceBuilder.CreateDefault()
+    .AddService(serviceName: serviceName, serviceVersion: serviceVersion);
+
+builder.Services.AddOpenTelemetryTracing(tracerProviderBuilder =>
+{
+  tracerProviderBuilder
+      .AddConsoleExporter()
+      .AddOtlpExporter(opt =>
+        {
+          opt.Protocol = OtlpExportProtocol.Grpc;
+          opt.Endpoint = new Uri("http://otel-collector:4317");

```

Comment:
I think we need to add configs for max batch size and timeout so it works fast

Reply:
Could we also add a commented-out section that uses `.http` with otel on port `4318`? Just to make sure people know when to use which. Also writing comments pointing this out would be awesome.

Reply:
Done and done 😄